core/src/components/picker-column/picker-column.tsx

```diff
@@ -412,15 +411,25 @@ export class PickerColumn implements ComponentInterface {
   };
 
   get activeItem() {
-    // If the whole picker column is disabled, the current value should appear active
-    // If the current value item is specifically disabled, it should not appear active
-    const selector = `.picker-item[data-value="${this.value}"]${this.disabled ? '' : ':not([disabled])'}`;
+    const { value } = this;
```

Comment:
We used to encode the value on the button element itself, but we don't do that anymore. As a result, we need to iterate through the components and check the value/disabled properties.